lib/runner/extensions/event.command.js

```diff
@@ -28,6 +33,56 @@ var _ = require('lodash'),
             error.name = ASSERTION_FAILURE;
         }
         return error ? serialisedError(error, true) : undefined;
+    },
+
+    /**
+     * Removes files in Request body if any.
+     *
+     * @param {Request~definition} request Request JSON representation to be sanitized
+     * @param {Function} callback function invoked with error and sanitizedRequest.
+     *                            sanitizedRequest is `null` if no files are detected.
+     */
+    sanitizeFiles = function (request, callback) {
```

Comment:
@shamasis I have a question.

If callback always gets the request, raw and mutated, how is that different from the synchronous version with the return?

Also, the signature of `affectedEntries` is not always an `Array`, it could the entire body for `file` model. If I had to send the raw request and affected entries, and let the callback to the reconstruction of new request then `sanitizeFiles` would not do much and it leaks concerns, right?